.github/workflows/run_update_exposure.yml

```diff
@@ -35,12 +33,13 @@ jobs:
         python pipelines/update_exposure.py
 
     - name: Trigger raster stats pipeline
-      uses: actions/github-script@v5
-      with:
-        github-token: ${{ secrets.PAT }}
-        script: |
-          await github.repos.createDispatchEvent({
-              owner: 'OCHA-DAP',
-              repo: 'ds-floodexposure-monitoring',
-              event_type: 'trigger_exposure_raster_stats'
-          })
+      env:
+        GITHUB_TOKEN: ${{ secrets.PAT }}
+      run: |
+        curl -L \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
```

Comment:
Darn, this looks like a tough one...

Reply:
Ah it's just taken from the [GH Action REST API documentation](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event).